next.clayui.com/content/docs/components/toolbars/management-toolbar.mdx

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
@diegonvs I'm thinking of moving this document out of `toolbars` since we only have it inside the folder. What do you think?

Reply:
Makes sense, for now, we don't have other Toolbar like component.